### DISCUSSION TITLE
Using the BVTree with TileCache Navmeshes

### DISCUSSION THREAD

**Discussion by SpangMcgraw** [recastnavigation/recastnavigation#685] [General]
By default tiled meshes use the BVTree. 

I wondered why the BVTree isn't used with TileCache meshes?

Is it purely a performance issue of updating the tree?

**Comment by memononen** [comment]
The tile cache meshes generally have far fever polygons, since the tiles are small, and also split vertically. That and the time it takes to compute the bv-tree.
